visual-dotnet/SauceLabs.Visual/VisualClient.cs

```diff
@@ -139,12 +139,29 @@ private async Task FinishBuild(VisualBuild build)
             return VisualCheckAsync(name, options);
         }
 
+        private void OrderIgnoredRegions(SelectiveRegion[]? regions, IgnoreRegion[]? ignoreRegions, IWebElement[]? ignoreElements, out RegionIn[] regionIns, out ElementIn[] elementIns)
```

Comment:
Maybe we can move it to another class to keep the client clean? We should assume the client code will be huge in the future.